projects/from-javascript-to-typescript/the-typeinator/01-syntactic-sugar/index.test.js

```diff
@@ -0,0 +1,22 @@
+const index = require("./index");
+const solution = require("./solution");
+const { announceMachines } = process.env.TEST_SOLUTIONS ? solution : index;
+
+describe("01 - Syntactic Sugar > announceMachines", () => {
```

Comment:
[Style] See other tests in this repository: the step names aren't part of the description titles.

```suggestion
describe("announceMachines", () => {
```